### ISSUE TITLE
[2][ahdigital]Convert Market link on assets into a button

### ISSUE THREAD

**Issue by mattbeckman** [bitshares/bitshares-ui#1073]
Let's improve this "Market" link. Minimally, give it margin. Optimally, make that baby look good.

<img width="691" alt="screen shot 2018-01-27 at 11 06 45 pm" src="https://user-images.githubusercontent.com/928380/35479890-365fbef2-03b7-11e8-863d-589d7251afdb.png">

**Comment by ahdigital** [comment]
No problem, I got this.

**Comment by ahdigital** [comment]
@wmbutler Are you happy for me to remove the text shown on the screenshot in red brackets for these views only?

![remove-text](https://user-images.githubusercontent.com/3088454/35559069-a187bcde-05a1-11e8-93a8-f26ac555164e.png)


It seems pointless to display it twice on the page.

**Comment by wmbutler** [comment]
I agree.

**Comment by ahdigital** [comment]
So when I visited this page from a mobile device it turned out that wasn't responsive either. Screenshot shows elements being cut off:

![asset-market-page](https://user-images.githubusercontent.com/3088454/35690494-ebec97ae-076d-11e8-959f-0a6181c36cf9.png)

Also the feed tables at the bottom of the page were unusable on mobile, and the tabs to switch between them didn't have a clear active state.

In summary, the actual changes completed are:

- Removed duplicated "issued by" text
- Converted "market" link into a button
- Fixed responsive layout
- Made feed tables swipe left and right on mobile
- Applied active state to tab

**Licecap example:**

![asset-market-page](https://user-images.githubusercontent.com/3088454/35690780-a8708f70-076e-11e8-8cd8-98cbb4ad3306.gif)

Will submit PR shortly.
